### DISCUSSION TITLE
RTMP Restream youtube

### DISCUSSION THREAD

**Discussion by temuccio** [blakeblackshear/frigate#15484] [General]
Hi all, 
it is possible restream a video or a birdseye to youtube via rtmp?
For example, now I use this command:
```
ffmpeg -loglevel info -threads 2 -filter_threads 2 \
       -thread_queue_size 512 \
       -i "<cameraRTSP>" \
       -f lavfi -i anullsrc=channel_layout=stereo:sample_rate=44100 \
       -pix_fmt yuv420p -c:v libx264 -preset veryfast -qp 19 -profile:v high -rc cbr \
       -level 4.2 -r 15 -g 60 -bf 3 -refs 3 \
       -c:a aac -b:a 128k \
       -f flv "rtmp://a.rtmp.youtube.com/live2/<key>"
```
At this page of documentation
https://docs.frigate.video/configuration/restream/
there is the chapter Advanced Restream Configurations...can I adjust the command with my ffmpeg command?

**Comment by NickM-27** [comment]
Go2rtc supports rtmp output, there's no reason to transcode the stream

**Reply by temuccio** [reply]
Hi @NickM-27, do you have an example of usage?
Thanks
